library_generation/generate_pr_description.py

```diff
@@ -97,10 +95,13 @@ def get_commit_messages(
     selecting commit message. This commit should be an ancestor of
     :param paths: a mapping from file paths to library_name.
     :param is_monorepo: whether to generate commit messages in a monorepo.
+    :param repo_level_message: commit messages regarding repo-level changes.
     :return: commit messages.
     :raise ValueError: if current_commit is older than or equal to
     baseline_commit.
     """
+    if current_commit_sha == baseline_commit_sha:
+        return EMPTY_MESSAGE
```

Comment:
The change could still come from library level change(e.g. api_description) even with the same commit_sha. Ideally we should combine all levels of changes, but I know this function might be only for generating repo level commit messages. If that's the case, can we rename the function?

Reply:
Library level change will not generate a pr description since we have a dedicated workflow to generate the description.

Reply:
Done.